myosuite/agents/sb3_job_script.py

```diff
@@ -10,12 +10,14 @@
 import os
 import json
 import time as timer
+import myosuite
```

Comment:
@cherylwang20 -- lets be sure that these additional imports are important

Reply:
removed this import, commit here [00e1751](https://github.com/MyoHub/myosuite/pull/192/commits/00e175112776fe21b7c0d9c52118ba895dde4956)